### PR TITLE
Update prices_base_tokens.sql

### PR DESCRIPTION
**Pull request by 0xOuterGod**
Added the following tokens: DEGEN, BVM, DAI+, USD+, PRIME, bTiUSD, YFX

# Thank you for contributing to Spellbook!

Thank you for taking the time to submit code in Spellbook. A few things to consider:
- If you are a first-time contributor, please sign the CLA by copy & pasting **_exactly_** what the bot mentions in PR comment
- Refer to [docs](#spellbook-contribution-docs) section below to answer questions
- Dune team will review submitted PRs as soon as possible

## Spellbook sub-projects

To prepare for Spellbook to scale in size, Dune team has started to pilot sub-projects:
- For a quick overview, see [the GH discussion on the topic](https://github.com/duneanalytics/spellbook/discussions/5238)
    - Please ask questions and continue the conversation there
- The first spell lineage to move: `tokens.erc20` and all upstream spells which help build it
    - **_Note_**: Please navigate to [tokens](https://github.com/duneanalytics/spellbook/tree/main/tokens) root directory to find tokens objects

## Spellbook contribution docs

The [docs](https://github.com/duneanalytics/spellbook/tree/main/docs) directory has been implemented to answer as many questions as possible. Please take the time to reference each `.md` file within this directory to understand how to efficiently contribute & why the repo is designed as it is 🪄

Example questions to be answered:
- What does each property in the [model config block](https://github.com/duneanalytics/spellbook/blob/main/docs/models/model_config_block.md) mean?
- What is the [CI test](https://github.com/duneanalytics/spellbook/blob/main/docs/ci_test/ci_test_overview.md) attached to PRs and how can I best utilize it?
- What are the Spellbook [best practices](https://github.com/duneanalytics/spellbook/blob/main/docs/general/best_practices.md)?

Please navigate through the [docs](https://github.com/duneanalytics/spellbook/tree/main/docs) directory to find as much info as you can.

**Note**: happy to take PRs to improve the docs, let us know 🤝

### CODE REVIEW

**Comment by Hosuke** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/duneanalytics/spellbook/blob/main/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> I have read the CLA Document and I hereby sign the CLA
> 
> You can retrigger this bot by commenting **recheck** in this Pull Request

Hi @0xOuterGod, please sign the CLA, so we can continue this PR.

**Comment by 0xOuterGod** [comment]
recheck

**Comment by Hosuke** [comment]
If you are a first-time contributor, please sign the CLA by copy & pasting exactly what the bot mentions in PR comment. Thank you.

**Comment by jeff-dude** [comment]
> If you are a first-time contributor, please sign the CLA by copy & pasting exactly what the bot mentions in PR comment. Thank you.

`I have read the CLA Document and I hereby sign the CLA`
in a stand-alone comment

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/prices/base/prices_base_tokens.sql [hunk 1]
```diff
@@ -28,5 +28,12 @@ FROM
     ('based-basedmarkets','base','BASED',0xba5e6fa2f33f3955f0cef50c63dcc84861eab663,18),
     ('usdc-usd-coin', 'base', 'USDC', 0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913, 6),
     ('wsteth-wrapped-liquid-staked-ether-20', 'base', 'wstETH',0xc1CBa3fCea344f92D9239c08C0568f6F2F0ee452,18),
-    ('aero-aerodrome-finance', 'base', 'AERO', 0x940181a94A35A4569E4529A3CDfB74e38FD98631, 18)
+    ('aero-aerodrome-finance', 'base', 'AERO', 0x940181a94A35A4569E4529A3CDfB74e38FD98631, 18),
+    ('degen-token', 'base', 'DEGEN', 0x4ed4e862860bed51a9570b96d89af5e1b0efefed,18),
```
Comment (jeff-dude) [inline models/prices/base/prices_base_tokens.sql:32]: https://api.coinpaprika.com/v1/coins/degen-token

Comment (jeff-dude) [reply models/prices/base/prices_base_tokens.sql:32]: id is invalid, please update to valid id (can find in url of token on coinpaprika)

**Review by jeff-dude — CHANGES_REQUESTED** [review]

**Comment by 0xOuterGod** [comment]
I have read the CLA Document and I hereby sign the CLA

File: models/prices/base/prices_base_tokens.sql (hunk 1 above)
Comment (0xOuterGod) [reply models/prices/base/prices_base_tokens.sql:32]:
```suggestion
    ('degen-degen-base', 'base', 'DEGEN', 0x4ed4e862860bed51a9570b96d89af5e1b0efefed,18),
    ('ftw-friendtech33', 'base', 'FTW', 0x3347453ced85bd288d783d85cdec9b01ab90f9d8,9),
```

Comment (0xOuterGod) [reply models/prices/base/prices_base_tokens.sql:32]: Now it should work. Added also another token

File: models/prices/base/prices_base_tokens.sql [hunk 2]
```diff
@@ -28,5 +28,13 @@ FROM
     ('based-basedmarkets','base','BASED',0xba5e6fa2f33f3955f0cef50c63dcc84861eab663,18),
     ('usdc-usd-coin', 'base', 'USDC', 0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913, 6),
     ('wsteth-wrapped-liquid-staked-ether-20', 'base', 'wstETH',0xc1CBa3fCea344f92D9239c08C0568f6F2F0ee452,18),
-    ('aero-aerodrome-finance', 'base', 'AERO', 0x940181a94A35A4569E4529A3CDfB74e38FD98631, 18)
+    ('aero-aerodrome-finance', 'base', 'AERO', 0x940181a94A35A4569E4529A3CDfB74e38FD98631, 18),
+    ('degen-degen-base', 'base', 'DEGEN', 0x4ed4e862860bed51a9570b96d89af5e1b0efefed,18),
+    ('ftw-friendtech33', 'base', 'FTW', 0x3347453ced85bd288d783d85cdec9b01ab90f9d8,9),
```
Comment (Hosuke) [inline models/prices/base/prices_base_tokens.sql:33]: This token is not active:
https://api.coinpaprika.com/v1/coins/ftw-friendtech33

We may need to remove `FTW` from this file.
```suggestion
```

File: models/prices/base/prices_base_tokens.sql [hunk 3]
```diff
@@ -28,5 +28,12 @@ FROM
     ('based-basedmarkets','base','BASED',0xba5e6fa2f33f3955f0cef50c63dcc84861eab663,18),
     ('usdc-usd-coin', 'base', 'USDC', 0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913, 6),
     ('wsteth-wrapped-liquid-staked-ether-20', 'base', 'wstETH',0xc1CBa3fCea344f92D9239c08C0568f6F2F0ee452,18),
-    ('aero-aerodrome-finance', 'base', 'AERO', 0x940181a94A35A4569E4529A3CDfB74e38FD98631, 18)
+    ('aero-aerodrome-finance', 'base', 'AERO', 0x940181a94A35A4569E4529A3CDfB74e38FD98631, 18),
+    ('degen-degen-base', 'base', 'DEGEN', 0x4ed4e862860bed51a9570b96d89af5e1b0efefed,18),
+    ('base-velocimeter', 'base', 'BVM', 0xd386a121991e51eab5e3433bf5b1cf4c8884b47a,18),
```
Comment (Hosuke) [inline models/prices/base/prices_base_tokens.sql:33]: Hi @0xOuterGod,
this token-id `base-velocimeter` is invalid.

Please verify.

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: models/prices/base/prices_base_tokens.sql (hunk 3 above)
Comment (0xOuterGod) [reply models/prices/base/prices_base_tokens.sql:33]: Verified, it's not available on Coinpaprika.

Comment (Hosuke) [inline models/prices/base/prices_base_tokens.sql:33]:
```suggestion
```

File: models/prices/base/prices_base_tokens.sql [hunk 4]
```diff
@@ -28,5 +28,11 @@ FROM
...
     ('usdc-usd-coin', 'base', 'USDC', 0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913, 6),
     ('wsteth-wrapped-liquid-staked-ether-20', 'base', 'wstETH',0xc1CBa3fCea344f92D9239c08C0568f6F2F0ee452,18),
-    ('aero-aerodrome-finance', 'base', 'AERO', 0x940181a94A35A4569E4529A3CDfB74e38FD98631, 18)
+    ('aero-aerodrome-finance', 'base', 'AERO', 0x940181a94A35A4569E4529A3CDfB74e38FD98631, 18),
+    ('degen-degen-base', 'base', 'DEGEN', 0x4ed4e862860bed51a9570b96d89af5e1b0efefed,18),
+    ('dai-plus-overnight', 'base', 'DAI+', 0x65a2508c429a6078a7bc2f7df81ab575bd9d9275,18),
+    ('usd-plus-overnight', 'base', 'USD+', 0xb79dd08ea68a908a97220c76d19a6aa9cbde4376,18),
+    ('prime', 'base', 'PRIME', 0xfa980ced6895ac314e7de34ef1bfae90a5add21b,18),
+    ('btiusd', 'base', 'bTiUSD', 0x82fdaf19c7560be8f29d4bb3a1f7e1620db27238,18),
+    ('yieldfarming-index', 'base', 'YFX'. 0x8901cb2e82cc95c01e42206f8d1f417fe53e7af0,18)
```
Comment (Hosuke) [inline models/prices/base/prices_base_tokens.sql:37]: These 3 tokens seem to be unavailable on coinpaprika.
I am going to remove these 3 tokens for now.
You can add back them with their valid ids in a new PR.
Thank you.

```suggestion
```

File: models/prices/base/prices_base_tokens.sql [hunk 5]
```diff
@@ -28,5 +28,8 @@ FROM
     ('based-basedmarkets','base','BASED',0xba5e6fa2f33f3955f0cef50c63dcc84861eab663,18),
     ('usdc-usd-coin', 'base', 'USDC', 0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913, 6),
     ('wsteth-wrapped-liquid-staked-ether-20', 'base', 'wstETH',0xc1CBa3fCea344f92D9239c08C0568f6F2F0ee452,18),
-    ('aero-aerodrome-finance', 'base', 'AERO', 0x940181a94A35A4569E4529A3CDfB74e38FD98631, 18)
+    ('aero-aerodrome-finance', 'base', 'AERO', 0x940181a94A35A4569E4529A3CDfB74e38FD98631, 18),
+    ('degen-degen-base', 'base', 'DEGEN', 0x4ed4e862860bed51a9570b96d89af5e1b0efefed,18),
+    ('dai-plus-overnight', 'base', 'DAI+', 0x65a2508c429a6078a7bc2f7df81ab575bd9d9275,18),
+    ('usd-plus-overnight', 'base', 'USD+', 0xb79dd08ea68a908a97220c76d19a6aa9cbde4376,18),
```
Comment (Hosuke) [inline models/prices/base/prices_base_tokens.sql:34]:
```suggestion
    ('usd-plus-overnight', 'base', 'USD+', 0xb79dd08ea68a908a97220c76d19a6aa9cbde4376,18)
```

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @0xOuterGod

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**